R/dp_init.R

```diff
@@ -77,10 +77,13 @@ dp_init <- function(project_path = fs::path_wd(),
   commit_description <- "dp init"
   wd0 <- fs::path_wd()
 
+  if (is_dp_initiated(path = fs::path_norm(fs::path(project_path, "..")), checks = c("git", "renv", "dp"))){
```

Comment:
The definition of `is_dp_initiated` does not take `checks` as a parameter. If so, checks need to be removed. Does this not throw an error?